### DISCUSSION TITLE
问下关于监听 touchstart 的作用

### DISCUSSION THREAD

**Discussion by IronKinoko** [ant-design/ant-design-mobile#5118] [Q&A] [answered]
问下面这段代码有什么说法吗？我目前的猜测是解决300ms点击延迟

```js
if (canUseDom) {
  document.addEventListener('touchstart', () => {}, true)
}
```

**Comment by GOWxx** [comment]
https://mobile.ant.design/zh/guide/quick-start
底部有钉钉群的二维码，如果愿意的话可以加一下，可能有更多的人看到你的消息并与你交流

**Comment by awmleer** [comment, marked as answer]
是为了让 iOS Safari 下 button 和 a 的 `:active` CSS 能生效，的确有点黑

更多信息可以看这里：https://stackoverflow.com/questions/3885018/active-pseudo-class-doesnt-work-in-mobile-safari

**Reply by miracles1919** [reply]
感觉可以加个注释 ~

**Reply by awmleer** [reply]
确实……我加一下
